README.md

```diff
@@ -83,13 +83,20 @@ the REST Proxy running using the default settings and some topics already create
 Development
 -----------
 
-To build a development version, you may need a development versions of
+To build a development version, you may need development versions of
 [common](https://github.com/confluentinc/common),
 [rest-utils](https://github.com/confluentinc/rest-utils), and
 [schema-registry](https://github.com/confluentinc/schema-registry).  After
 installing these, you can build the Kafka REST Proxy
 with Maven. All the standard lifecycle phases work.
 
+If your contribution will be a patch or feature and target the latest or earlier
+release version, you can avoid building development versions of dependencies
```

Comment:
I agree, it reads better without the initial waffle ...

No, all code-contributions can't be covered by this shortcut, and anything on a pre-release, development branch won't work, as evidenced by the earlier tests on `master`, `6.2.x` et. al.  I expect these would still require upstream deps to be built to a local repo as a specific version.

This is a compromise - contributors need access to be able to build branches with only access to public repositories - currently this is satisfied by any release tag, or any `-post` branch, as they have dep versions locked to those in public repos.  Now if the contribution is minor, or a bugfix/patch, or if it only relies on released code and deps, then it might be reasonable to allow that code to be developed with such a tag or `-post` branch as a base.  That's the only simple route I see available, as working on a pre-release branch will require pre-building of all the deps.